docs/DIRECTORY_STRUCTURE.md

```diff
@@ -1,24 +1,27 @@
-# Project Directory Structure
+# Directory Structure
 
 Welcome to our project! This document outlines our project's directory structure, adhering to a "Feature-Driven Folder Structure" approach. The structure is inspired by insights from both an [English article](https://dev.to/profydev/screaming-architecture-evolution-of-a-react-folder-structure-4g25#indexjs-as-public-api) and a [Japanese article](https://zenn.dev/necscat/articles/d5d9b7a3f859d7).
 
-## Directory Structure Overview
+## Overview
 
-- [Project Directory Structure](#project-directory-structure)
-  - [Directory Structure Overview](#directory-structure-overview)
+- [Directory Structure](#directory-structure)
+  - [Overview](#overview)
     - [`app/`](#app)
```

Comment:
Is it not needed to mention inside of `app/` directory ?

Reply:
This directory is for routing, and all the other sub-directories are corresponded to the URL paths. Therefore, they are really not suitable for writing here.